content/en/quickstart/create-device.md

```diff
@@ -11,3 +11,105 @@ menu:
     weight: 10
 toc: true
 ---
+This guide will cover how to get the first steps while using the plgd cloud in the Raspberry Pi. This includes how to create cloud server (OCF Device) available from IoTivity-Lite open source project on the Github.
```

Comment:
This guide will walk you through the process of starting your secure [OCF Device](link to spec - what is it, probably somewhere in core) using [IoTivity](.../iotivity-lite) library on the Raspberry Pi.